CONTRIBUTING.md

```diff
@@ -4,6 +4,7 @@
 To adding a new api, for example TokenizationAPI
 - Update Open API specification in `specs` folder to latest.
 - Generate request, response models by run the script in `scripts/generate.sh`
+- Find and replace `app_id`, `payment_id`, `mac`, `sig` in the generated request files
```

Comment:
We should make clear that these fields must be modified to optional 
Add a question mark after these fields